cmd/aws-k8s-tester/eks/create-stresser-v2.go

```diff
@@ -0,0 +1,595 @@
+package eks
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-k8s-tester/pkg/randutil"
```

Comment:
Can we separate std package imports and 3rd party dependencies and run `./hack/fmt.sh`?

```
"context"
...

"github.com/..."
```

Reply:
yep